pkg/graph/resolvers/itgovactions/newstep/progress_to_new_step.go

```diff
@@ -68,6 +74,10 @@ func UpdateIntake(intake *models.SystemIntake, newStep models.SystemIntakeStepTo
 	case models.SystemIntakeStepToProgressToGrbMeeting:
 		intake.Step = models.SystemIntakeStepGRBMEETING
 
+		if grbReviewType != nil {
+			intake.GrbReviewType = grbReviewType
+		}
```

Comment:
I want to confirm this is the appropriate way to update this field. In MINT, we generally treat nil inputs as a NOOP, but I don't know if that's the intended behavior here. Conversely we can update the GQL such that this is a mandatory field.